src/cmake/thirdparty/SetupConduit.cmake

```diff
@@ -93,7 +93,7 @@ endif()
 set(CONDUIT_FOUND TRUE)
 set(CONDUIT_INCLUDE_DIRS ${CONDUIT_DIR}/include/conduit)
 
-if(PYTHON_FOUND)
+if(ENABLE_PYTHON)
```

Comment:
Just a note: i don't think we are consistent in `ENABLE_X`  versus `FOUND_X`

Reply:
I agree, in this case, I need to change b/c it was being stubborn, I was mixing builds of TPLS. Since not being able to override caused an error, I decided that we need the extra control of ENABLE_PYTHON